functions/__sf_section_ruby.fish

```diff
@@ -29,6 +29,8 @@ function __sf_section_ruby -d "Show current version of Ruby"
 		set ruby_version (rvm-prompt i v g)
 	else if type -q rbenv
 		set ruby_version (rbenv version-name)
+  else if type -q chruby
+    set ruby_version (echo $RUBY_AUTO_VERSION)
```

Comment:
Also, I assume `$RUBY_AUTO_VERSION` exists automatically when `chruby` is installed? [spaceship's implementation](https://github.com/denysdovhan/spaceship-prompt/blob/master/sections/ruby.zsh#L33) has a different way to go about this, but this seems clearer if it works 😄

Reply:
Oh that way works too, weird that line hasn't changed in 5 years though, so not sure why that's in there. Checking to see if it's set automatically on install now